docs/apm/traces/working-with-tracing-data/tracing-dashboards.md

```diff
@@ -5,132 +5,98 @@ sidebar_label: Tracing Dashboards
 description: Create Tracing dashboards, install apps, and more.
 ---
 
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
 ## Application Service Dashboards
 
 Service and Application dashboards are available as two Explore hierarchies. You can access them in the **Explore By** drop down menu.
 
-The **Application Service View** groups services into higher-level applications based on the `application=[app-name]` custom tag that you may add to your tracing data if you want to leverage it fully. If not present, all services will belong to a "default" application. See [how to modify the attributes of your spans](../advanced-configuration/working-with-span-attributes.md). The third level shows the top 50 most active operations executed on the service.
+The **APM: Application View** groups services into higher-level applications based on the `application=[app-name]` custom tag that you may add to your tracing data if you want to leverage it fully. Without the tag, all services will belong to a "default" application. The third level shows the top 50 most active operations executed on the service. See [Working with Span attributes](../advanced-configuration/working-with-span-attributes.md) to learn how to how to customize your span attributes.
```

Comment:
The **APM: Application View** groups services into higher-level applications based on the `application=[app-name]` custom tag that you may add to your tracing data if you want to leverage it fully. Without the tag, all services will belong to a "default" application. The third level shows environment automatically picked from OT deployment.environment tag. Without the tag, all data will belong to a "default" environment. 
The fourth and last shows the top 50 most active operations executed on the service. See [Working with Span attributes](../advanced-configuration/working-with-span-attributes.md) to learn how to how to customize your span attributes.